xbmc/addons/addoninfo/AddonInfo.h

```diff
@@ -47,6 +47,14 @@ enum class AddonOriginType
   MANUAL = 2 /// The addon origin is a zip file, package or development build
 };
 
+enum class AddonUpdateRule
+{
+  /// @brief Reasons why an addon is not updateable
+  ANY = 0, /// used internally, not to be explicitly set
+  USER_DISABLED_AUTO_UPDATE = 1, /// automatic updates disabled via AddonInfo dialog
+  PIN_OLD_VERSION = 2 /// user downgraded to an older version
```

Comment:
If you make a doc behind a value must be for doxygen arrow there, also must be the brief outside.

```cpp
//! @brief Reasons why an addon is not updateable
enum class AddonUpdateRule
{
  ANY = 0, //!< used internally, not to be explicitly set
  USER_DISABLED_AUTO_UPDATE = 1, //!< automatic updates disabled via AddonInfo dialog
  PIN_OLD_VERSION = 2 //!< user downgraded to an older version
};
```

Try also to use inside Kodi for doxygen `/** ... */`, `/*! ... */` or `//! ...`.
The `///` I prever to use for addon API (binary/python) to be able to tell these two parts apart.

Reply:
will fix the comments shortly... 👍

Reply:
Also overseen with my bad english the **`updateable`** should be **`updatable`** :sweat_smile:

Reply:
one minute too late @AlwinEsch :)
read @ stackexchange _Both of them are correct and acceptable._
if our native english speakers insist on `updatable` i will include this in the refactor that may come after this pr

comments changed otherwise.. nothing else that was just the rebase

Reply:
Both are valid spellings.